tests/CoreFX/TopN.CoreFX.Windows.issues.json

```diff
@@ -0,0 +1,3686 @@
+[
```

Comment:
Would it be possible to run all tests by default, and just have (hopefully small) file with disabled tests?

This 3500+ lines file will be constantly outdated as new tests are added in corefx.

Reply:
Certainly - it would be a straightforward change here. But the JSON list in blob storage holds a list of all tests in the blob container, so outdated tests will still be an issue.

Alternatively we could have the URL point to a list of artifacts from a Helix build (automatically generated), but that would require CoreFX to move to the updated version of the XUnit runner and would need also need to be updated as the URLs become stale.